.github/workflows/release.yaml

```diff
@@ -157,3 +157,38 @@ jobs:
           git add -f _data/registry/releases/$FILENAME.json    #ignored due to .gitignore config
           git commit -m "Automated Update For Apicurio Registry Release Version: ${{steps.metadata.outputs.release-version}}"
           git push
+
+  verify-release:
+    runs-on: ubuntu-18.04
+    needs: ["release"]   # The Job gets triggered only after the "release" job has successfully completed. The job doesn't run in case the "release" job fails
```

Comment:
@EricWittmann In general, if there are multiple jobs present within a single GitHub action workflow, then all the job runs in parallel. However, we can control this behavior using the `needs` keyword. It basically accepts an array where we can specify the job-names. This will prevent the job from running parallelly with other jobs, instead it will wait for the specified jobs to complete and will start running only if the specified jobs have completed successfully.

Reply:
Also, I decided not to have a separate workflow for this because the step that verifies the maven deploy requires us to pass the release version at runtime. We fetch this information from project.yaml. However, the GitHub action that we use to fetch the metadata from the 'project.yaml' file works only with the pull request event. This would mean that the trigger for the workflow must be the pull-request event. Another thing is, on the pull request event, both the release and verify-release workflow will get triggered simultaneously. But we want the 'verify-release' workflow to get triggered only after the release has been made otherwise it will fail.

Reply:
Thanks for the extra info, @riprasad !